CMakeLists.txt

```diff
@@ -1,49 +0,0 @@
-cmake_minimum_required(VERSION 3.18.1)
-project(odr-droid CXX)
-
-set(CMAKE_CXX_STANDARD 17)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-# create virtual environment in project root and "pip install conan==1.*"
-set(CMAKE_PREFIX_PATH "${CMAKE_CURRENT_LIST_DIR}/bin")
```

Comment:
how can we make this easier for users? I kept this comment in as a reminder for myself how to make it work in an ugly way, but there's probably a clean way to do it without installing global dependencies

Reply:
I removed it for two reasons
- conan v1
- cmake is now called after conan which is also called by gradle

I think it is easiest to prefix the path of the conan executable here https://github.com/opendocument-app/OpenDocument.droid/blob/ef42f76f2822222cabf4502af01e514a26b06113/app/build.gradle#L25